blueprint-test-utils/src/lib.rs

```diff
@@ -551,4 +551,99 @@ mod tests_standard {
             })
             .await
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_eigen_incredible_squaring_blueprint() {
+        let mut base_path = std::env::current_dir().expect("Failed to get current directory");
+
+        base_path.push("../blueprints/incredible-squaring");
+        base_path
+            .canonicalize()
+            .expect("File could not be normalized");
+
+        let manifest_path = base_path.join("Cargo.toml");
+        let blueprint_test_file = base_path.join("blueprint-test.json");
+        let blueprint_file = base_path.join("blueprint.json");
+
+        // cp the blueprint-test-file into the blueprint-file
+        tokio::fs::copy(&blueprint_test_file, &blueprint_file)
+            .await
+            .expect("Failed to copy blueprint-test.json to blueprint.json");
+
+        let opts = Opts {
+            pkg_name: Some("incredible-squaring-blueprint".to_string()),
+            rpc_url: "ws://127.0.0.1:9944".to_string(),
+            manifest_path,
+            signer: None,
+            signer_evm: None,
+        };
+        // --ws-external
+        const INPUT: u64 = 10;
+        const OUTPUT: u64 = INPUT.pow(2);
+
+        // Start Local Anvil Testnet in the background
+        let _ = eigensdk_rs::test_utils::anvil::testnet::incredible_squaring::run_incredible_squaring_testnet().await;
+
+        new_test_ext_blueprint_manager::<5, 1, (), _, _>((), opts, run_test_blueprint_manager)
+            .await
+            .execute_with_async(move |client, handles, logger| async move {
+                // At this point, blueprint has been deployed, every node has registered
+                // as an operator for the relevant services, and, all gadgets are running
+
+                // What's left: Submit a job, wait for the job to finish, then assert the job results
+                let keypair = handles[0].sr25519_id().clone();
+
+                // TODO: Important! The tests can only run serially, not in parallel, in order to not cause a race condition in IDs
```

Comment:
Can we explain this more? How can we get to parallel test execution? We want to mimic the real-world as close as possible if we possibly can.